src/common/components/Select/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { defaultValues } from '@constants/defaultValues';
+import { IconChevronDown } from '@sopt-makers/icons';
+import { ChangeEvent } from 'react';
+
+import {
+  circle,
+  containerVar,
+  error,
+  icon,
+  optionContainer,
+  optionLabel,
+  selectContainer,
+  selectVariant,
+  title,
+} from './style.css';
+import { SelectBoxProps } from './type';
+
+const SelectBox = ({ label, options, size = 'sm', required, formObject }: SelectBoxProps) => {
+  const { register, setValue, formState, clearErrors } = formObject;
+  const { dirtyFields, errors } = formState;
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    clearErrors && clearErrors(label);
+    setValue(label, e.currentTarget.id, { shouldValidate: true, shouldDirty: true });
+  };
+
+  return (
+    <div className={containerVar[size]}>
+      <label className={title} htmlFor={label}>
+        <span>{label}</span>
+        {required && <i className={circle} />}
+      </label>
+      <div className={selectContainer}>
+        <input
+          type="button"
+          className={selectVariant[errors?.[label] ? 'error' : dirtyFields[label] ? 'selected' : 'default']}
+          role="combobox"
+          onFocus={() => clearErrors && clearErrors(label)}
+          {...register(label, { validate: (v) => v !== defaultValues[label] || '필수 입력 항목이에요.' })}
+        />
```

Comment:
defaultValues와 비교를 하는데 아무것도 선택을 안 할 경우에도 defaultValue와 다른 것으로 간주되어 required 상태가 아니더라도 항상 값을 입력해야 하는 상황이 발생하는 거 같습니다!

```suggestion
          {...register(label, { validate: (v) => !required || v !== defaultValues[label] || '필수 입력 항목이에요.' })}
        />
```
이렇게 수정하면 이러한 상황을 예방할 수 있을 거 같아요!

Reply:
완전 감사합니다~~